app/src/main/res/layout/fragment_event_details.xml

```diff
@@ -497,6 +497,24 @@
 
                     </FrameLayout>
 
+                     <!-- Added-->
+                    <FrameLayout
+                        android:id="@+id/event_owner_detail_fragment"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        bind:layout_constraintEnd_toEndOf="parent"
+                        bind:layout_constraintStart_toStartOf="parent"
+                        bind:layout_constraintTop_toBottomOf="@+id/event_statistics_frame">
+
+                        <include
+                            android:id="@+id/event_owner_detail"
+                            layout="@layout/event_owner_detail"
+                            bind:eventOwner="@{event}"
+                             />
+
+                    </FrameLayout>
+                    <!--Added-->
```

Comment:
Remove useless comments and format XML correctly

Reply:
OK @iamareebjamal Is this layout good or should I improve more.

Reply:
Good